Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM python:3.7-slim-buster AS base
 EXPOSE 5000
-WORKDIR /todo-app
+WORKDIR /todo_app
```

Comment:
Be careful with this change, it feels fairly subtle, and not in keeping with your previous approach!

For example, I think it will break your production container since that relies on the existing folder structure:
```
COPY ./todo_app /todo-app/todo_app
```

Looking at the above & the bind mount you specified for the development container, you previously set the app up to have `/todo-app` as the root folder and `/todo-app/todo_app` as the code folder. Sticking with that convention feels sensible for your test container therefore, so I'd suggest switching this back and then having the below (at lines 20/21 in this file, i.e. as part of the test stage) so that you keep the same structure as you have in your Git repo (rather than having it differ between your Git repo and Docker)

```
COPY ./todo_app ./todo_app
COPY ./test_to_do_app ./test_to_do_app
```